pkg/action/executor/resize_container.go

```diff
@@ -131,7 +143,16 @@ func (r *ContainerResizer) buildResourceLists(pod *k8sapi.Pod, actionItem *proto
 	//1. check capacity change
 	change, amount := getNewAmount(comm1.GetCapacity(), comm2.GetCapacity())
 	if change {
-		if err := r.buildResourceList(pod, cType, amount, spec.NewCapacity); err != nil {
+		var resourceList k8sapi.ResourceList
+		if _, exists := resourceCommodities[cType]; exists {
+			resourceList = spec.NewCapacity
+		} else if _, exists := resourceRequestCommodities[cType]; exists {
+			resourceList = spec.NewRequest
+		} else {
+			return fmt.Errorf("failed to build resource list when resize %v capacity to %v: %v commodity type is not supported",
```

Comment:
nit: I think, if you use %s you can directly use the variable name implementing `String()` method.
Additionally %v (print golang value representation) is recommended when you don't know, what you want to print first hand.
```
fmt.Errorf("failed to build resource list when resize %s capacity to %v: %s commodity type is not supported",
cType, amount, cType)
```

Reply:
Thanks @irfanurrehman. Will update the message using %s here